### PR TITLE
ci: move E2E runner from github to AWS

### LINKED ISSUE #104: Convert E2E job to use AWS runners
See https://github.com/instructlab/instructlab/issues/2036 for reference

### PR DESCRIPTION
**Pull request by nathan-weinberg**
Resolves https://github.com/instructlab/eval/issues/104

### CODE REVIEW

**Comment by alimaredia** [comment]
@nathan-weinberg can you send a run triggered by hand that shows that these new runners work?

**Review by alimaredia — APPROVED** [review]

**Comment by alimaredia** [comment]
Ack, in the past there was another repo we used to test new runners. But we can make modifications as needed.

File: .github/workflows/e2e-nvidia-t4-x1.yml [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+        working-directory: "${{ env.GITHUB_WORKSPACE }}/eval"
+        run: |
+          . "${{ env.GITHUB_WORKSPACE }}/instructlab/venv/bin/activate"
+          pip install .
+          pip install .[cuda]        
+
+      - name: Run e2e test
+        run: |
+          . venv/bin/activate
+          ./instructlab/scripts/basic-workflow-tests.sh -m
```
[the remark this answers is not in the document]
Comment (nathan-weinberg) [reply .github/workflows/e2e-nvidia-t4-x1.yml:133]: Right now this is matching what we know works on this size of runner, I think @alimaredia is doing some work to see what else we can squeeze out of this runner

**Review by danmcp — APPROVED** [review]

**Comment by nathan-weinberg** [comment]
Since the Mergify config changes to require an E2E run that hasn't begun running yet, need to merge this manually

**Pull request merged**